ext/ITensorNetworksEinExprsExt.jl

```diff
@@ -0,0 +1,37 @@
+module ITensorNetworksEinExprsExt
+
+using ITensorNetworks
+using ITensorNetworks: Index
+using EinExprs: EinExprs, EinExpr, einexpr, SizedEinExpr
+
+function ITensorNetworks.contraction_sequence(
+  ::ITensorNetworks.Algorithm"einexpr",
+  tn::ITensorNetwork{T};
+  optimizer=EinExprs.Exhaustive(),
+) where {T}
+  IndexType = Any
+
+  tensors = EinExpr{IndexType}[]
+  tensor_map = Dict{Set{IndexType},T}()
+  sizedict = Dict{IndexType,Int}()
+
+  for (key, tensor) in pairs(vertex_data(tn))
+    _inds = collect(ITensorNetworks.inds(tensor))
```

Comment:
I think `inds()` should just work as it is exported by `ITensors`.

Reply:
The problem is that `EinExprs` also exports `inds`, so I put it to make sure which `inds` function it's using.

Reply:
`tensor` here is presumably an `ITensor` type? Is `EinExprs.inds()` defined for an `ITensor` too? If so then I understand keeping `ITensorNetworks.inds`

Reply:
Yes, here `tensor` is an `ITensor`.
Nope, `EinExprs.inds` is not defined for `ITensor`.

Reply:
I see, `ITensors` exports `inds` so shouldn't `inds(it::ITensor)` just work without the prefix as there is no ambiguity with `EinExprs`?